host/xtest/regression_4000.c

```diff
@@ -5436,6 +5436,9 @@ static void xtest_tee_test_4011(ADBG_Case_t *c)
 
 		/* 4.3 */
 		n = n + i + tmp_size - m;
+		if (!ADBG_EXPECT_COMPARE_UNSIGNED(c, n, <=, tmp_size))
```

Comment:
Please add a comment explaining when this is needed, compare with the comment above around line 5426.